mint-auditor/src/bin/main.rs

```diff
@@ -85,7 +94,8 @@ pub struct Config {
     pub command: Command,
 }
 
-fn main() {
+#[rocket::main]
```

Comment:
I think that you can use #[launch] instead? This [documentation](https://api.rocket.rs/v0.5-rc/rocket/attr.main.html) seems to suggest that #[rocket::main] is intended only when you want to inspect ignite and launch.